overdrive.py

```diff
@@ -16,12 +17,12 @@ class OverdriveBibliographicCoverageProvider(
 
     EXCLUDE_SEARCH_INDEX = True
 
-    def __init__(self, uploader, collection, *args, **kwargs):
+    def __init__(self, collection, uploader=None, **kwargs):
```

Comment:
The provider couldn't run with the `RunCollectionCoverageProvider` without this parameter change.